cfgov/v1/jinja2/v1/includes/molecules/global-eyebrow.html

```diff
@@ -10,18 +10,26 @@
 
    is_horizontal: Whether the molecule appears horizontally or vertically.
                   defaults to false.
+
    language:      The page's language value (used to display different
                   information on Spanish pages). Defaults to English.
 
+   has_tagline:   Whether this eyebrow should include the tagline or not.
+
    ========================================================================== #}
-{% macro render( is_horizontal=false, language='en' ) %}
+{% macro render(
+    is_horizontal=false,
+    language='en',
+    has_tagline=true ) %}
 {% import 'v1/includes/atoms/tagline.html' as tagline with context %}
     <div class="m-global-eyebrow
                 m-global-eyebrow__{{ 'horizontal' if is_horizontal
                                       else 'list' }}">
         <div class="wrapper
                     {{ 'wrapper__match-content' if is_horizontal else '' }}">
-            {{ tagline.render() }}
+            {% if has_tagline %}
+                {{ tagline.render() }}
+            {% endif %}
```

Comment:
Maybe use Jinja syntax to collapse whitespace and avoid adding blank lines to the HTML?

```suggestion
            {% if has_tagline -%}
                {{ tagline.render() }}
            {%- endif %}
```